.storybook/preview.js

```diff
@@ -28,7 +30,23 @@ export const parameters = {
     ],
   },
   docs: {
-    theme: MetaMaskStorybookTheme,
+    container: (context) => {
+      const isDark = useDarkMode();
+
+      const props = {
+        ...context,
+        theme: isDark
+          ? { ...themes.dark, ...metamaskStorybookTheme }
+          : { ...themes.light, ...metamaskStorybookTheme },
```

Comment:
Updates colors as but adheres to brand fonts and some other small theme customizations